src/scopyApplication.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef SCOPYAPPLICATION_H
+#define SCOPYAPPLICATION_H
+
+#include <QtGlobal>
+
+#define __STDC_FORMAT_MACROS
+#ifdef Q_OS_WIN
+#include "breakpad/client/windows/handler/exception_handler.h"
+#endif
+
+#ifdef Q_OS_LINUX
+#include "client/linux/handler/exception_handler.h"
+#endif
+
+//#ifdef CATCH_UNHANDLED_EXCEPTIONS

```

Comment:
needed?

Reply:
Enabling this will catch unhandled exceptions but it seems OS-dependant. I'd keep this as it will make debug easier in the future.

Reply:
Actually, what I wanted to say was that I've seen this in the .cpp file as well and was wondering if is is needed in the header as well.

Reply:
This was supposed to be a  #define.